docs/source/tutorials/outliers.ipynb

```diff
@@ -516,10 +544,105 @@
     "</div>"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "id": "937c7e97",
+   "metadata": {},
+   "source": [
+    "## 5. Use cleanlab and `pred_probs` to find outliers in the dataset\n",
+    "\n",
+    "\n",
+    "Another way we can detect unusual exampels in the data is using prediction probabilities for out-of-distribution detection. Instead of using intermediate feature representations, this type of outlier detection simply needs a model's `pred_probs` to work. These methods are faster than using `feature_embeddings` but as a tradeoff perform slightly worse. This is due to the intermediate feature embeddings containing more auxillary information for every example, only some of which is represented in the final predicted probabilites.\n",
```

Comment:
typo: exampels

Reply:
generally want to say "predicted probabilities" or "predicted class probabilities" instead of "prediction probabilities"

Reply:
"as a tradeoff" -> "may"